README.md

```diff
@@ -43,22 +43,51 @@ To work with the quipucords code, begin by cloning the repository:
 git clone git@github.com:quipucords/quipucords.git
 ```
 
-quipucords currently supports Python 3.6. If you do not have Python on your system, follow these [instructions](https://www.python.org/downloads/).
+quipucords currently supports Python 3.9. If you do not have Python on your system, follow these [instructions](https://www.python.org/downloads/).
 
+## Setting Up a Tool to Manage Multiple Runtime Versions
+*asdf* is a single CLI tool and command interface that manages each of the project runtimes. It isn't mandatory, but highly recommended. You could alternatively install *pyenv* and *nvm*. For the instructions below we will assume you have it installed.
+
+See [asdf installation instructions](http://asdf-vm.com/guide/getting-started.html#_2-download-asdf) for more information.
```

Comment:
Just a very minor suggestion - perhaps the link should point to the first section in the asdf doc - http://asdf-vm.com/guide/getting-started.html#_1-install-dependencies